browse/services/documents/fs_implementation/legacy_fs_paths.py

```diff
@@ -12,7 +12,8 @@
 from .formats import list_ancillary_files
 from ..format_codes import has_ancillary_files, VALID_SOURCE_EXTENSIONS
 
-UPath = Union[CloudPath, Path]
+from browse.services import APath
```

Comment:
CloudPathLib has a function to_anyypath which returns either a Path for a file name like '/bobs/uncle' or a CloudPath for a name like 'gs://bobs/uncle'. This is a type to represent that.